prov/sm2/src/sm2_ep.c

```diff
@@ -297,6 +297,49 @@ static ssize_t sm2_do_inject(struct sm2_ep *ep, struct sm2_region *peer_smr,
 	return FI_SUCCESS;
 }
 
+static void sm2_format_cma(struct sm2_xfer_entry *xfer_entry,
+			   enum fi_hmem_iface iface, uint64_t device,
```

Comment:
you shouldn't need iface/device on the cma path

Reply:
done